CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.4.0] - 2015-09-15
```

Comment:
It's not released yet, so let's keep this section as `Unreleased` and no date for now